src/molecules/user/user.js

```diff
@@ -1,19 +1,31 @@
-import React from 'react';
-import Avatar from 'src/atoms/avatar';
-import Highlight from 'src/atoms/highlight';
-import { propTypes, defaultProps } from './prop-type';
-import UserMeta from './user-meta';
+import React from "react";
+import UserMeta from "./user-meta";
+import Avatar from "src/atoms/avatar";
+import Highlight from "src/atoms/highlight";
+import { propTypes, defaultProps } from "./prop-type";
+import { mentionLink } from "./util";
 import {
 	UserContainer,
 	UserAvatar,
 	UserName,
 	UserDescription,
-	UserDescriptionHighlight,
-} from './elements';
+	UserDescriptionHighlight
+} from "./elements";
 
 const decorators = {
-	'@': (segment, match, key) => <UserDescriptionHighlight key={key}>{segment}</UserDescriptionHighlight>,
-	'#': (segment, match, key) => <UserDescriptionHighlight key={key}>{match}</UserDescriptionHighlight>,
+	"@": (segment, match, key) => (
+		<UserDescriptionHighlight
+			key={key}
+			href={mentionLink(segment)}
```

Comment:
I must add a bit more documentation to the highlighter atom I think. In the `mentionLink` you separate the special character `@` from the text right? Luckily I thought about this use case when I built the highlighter, it's the `match` parameter 😄  But again, this is fully my fault for lack of documentation.

---

TL;DR; The `match` parameter is the value you can use here, without having to separate the character from text yourself 😄 

---

The decorator is invoked with the segment, match and key. Take this text for example:

```
Something something @abc somehting @qwe
```

This will result in 2x method invocations with the `@` decorator with the following arguments:

invocation | segment | match | key
---------- | ------- | ----- | ---
\# 0       | `@abc`  | `abc` | `0`
\# 1       | `@qwe`  | `qwe` | `1`